Gems/ScriptAutomation/Code/Source/ImageComparisonSettings.cpp

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <ImageComparisonSettings.h>
+#include <Atom/RPI.Reflect/Asset/AssetUtils.h>
+
+namespace AZ::ScriptAutomation
+{
+    void ImageComparisonSettings::Activate()
+    {
+        m_configAsset = AZ::RPI::AssetUtils::LoadAssetByProductPath<AZ::RPI::AnyAsset>("config/imagecomparisonconfig.azasset", AZ::RPI::AssetUtils::TraceLevel::Assert);
```

Comment:
This file need to be added to the gem's seed file so it would be packed in o3de binary build.
I wonder why use an asset to save the data instead of using setting registry

Reply:
The asset file is duplicated from ASV, as is this logic :)

Reply:
I added a seedList file to the gem for the data file.